packages/terra-site/src/App.jsx

```diff
@@ -60,36 +72,35 @@ class App extends React.Component {
     );
 
     const bidiContent = (
-      <ButtonGroup
-        className={styles['site-bidi']}
-        dir="ltr"
-        size="medium"
-        isSelectable
-        buttons={[
-          <ButtonGroup.Button isSelected text="ltr" key="ltr" onClick={() => document.getElementsByTagName('html')[0].setAttribute('dir', 'ltr')} />,
-          <ButtonGroup.Button text="rtl" key="rtl" onClick={() => document.getElementsByTagName('html')[0].setAttribute('dir', 'rtl')} />,
-        ]}
-      />
+      <CollapsibleMenuView.ItemGroup key="site-bidi" isSelectable dir="ltr" size="medium">
+        <CollapsibleMenuView.Item id="ltr" text="ltr" key="ltr" isSelected={this.state.dir === 'ltr'} onClick={this.handleBidiChange} />
+        <CollapsibleMenuView.Item id="rtl" text="rtl" key="rtl" isSelected={this.state.dir === 'rtl'} onClick={this.handleBidiChange} />
+      </CollapsibleMenuView.ItemGroup>
     );
 
     const localeContent = (
-      <div className={styles['site-locale']}>
-        <label htmlFor="locale"> Locale: </label>
-        <select value={this.state.locale} onChange={this.handleLocaleChange}>
-          <option value="en">en</option>
-          <option value="en-GB">en-GB</option>
-          <option value="en-US">en-US</option>
-          <option value="de">de</option>
-          <option value="es">es</option>
-          <option value="fr">fr</option>
-          <option value="pt">pt</option>
-          <option value="fi-FI">fi-FI</option>
-        </select>
-      </div>
+      <CollapsibleMenuView.Item
+        text={`Locale: ${this.state.locale}`}
+        key="locale"
+        menuWidth="160"
+        shouldCloseOnClick={false}
+        subMenuItems={[
+          <CollapsibleMenuView.ItemGroup isSelectable key="local-options">
+            <CollapsibleMenuView.Item id="en" text="en" key="en" isSelected={this.state.locale === 'en'} onClick={this.handleLocaleChange} />
+            <CollapsibleMenuView.Item id="en-GB" text="en-GB" key="en-GB" isSelected={this.state.locale === 'en-GB'} onClick={this.handleLocaleChange} />
+            <CollapsibleMenuView.Item id="en-US" text="en-US" key="en-US" isSelected={this.state.locale === 'en-US'} onClick={this.handleLocaleChange} />
+            <CollapsibleMenuView.Item id="de" text="de" key="de" isSelected={this.state.locale === 'de'} onClick={this.handleLocaleChange} />
+            <CollapsibleMenuView.Item id="es" text="es" key="es" isSelected={this.state.locale === 'es'} onClick={this.handleLocaleChange} />
+            <CollapsibleMenuView.Item id="fr" text="fr" key="fr" isSelected={this.state.locale === 'fr'} onClick={this.handleLocaleChange} />
+            <CollapsibleMenuView.Item id="pt" text="pt" key="pt" isSelected={this.state.locale === 'pt'} onClick={this.handleLocaleChange} />
+            <CollapsibleMenuView.Item id="fi-FI" text="fi-FI" key="fi-FI" isSelected={this.state.locale === 'fi-FI'} onClick={this.handleLocaleChange} />
+          </CollapsibleMenuView.ItemGroup>,
```

Comment:
You could simplify this a little more by using the onChange for the group instead of the onClick for each item.

Reply:
🙈 You're right. Good call.